torch/csrc/jit/passes/quantization.cpp

```diff
@@ -938,19 +954,30 @@ void InsertPrepackUnpack(script::Module& module) {
 void FoldPrepackedWeightIntoModule(
     script::Module& module,
     const std::string& method_name,
-    const script::Module& wrapper_module) {
+    const script::Module& linear_params_module,
+    const script::Module& conv_params_module) {
```

Comment:
This looks a bit ugly. Are `linear` and `conv` the only ops that might need packing? Should we consider making this more general?

Reply:
these two should be the only cases, and these are workaround, I think we'll have proper support before we have more use cases.